.github/actions/retry/action.yml

```diff
@@ -0,0 +1,23 @@
+name: "Retry Steps"
+inputs:
+  command:
+    required: true
+  working-directory:
+    required: false
```

Comment:
Should this be `true`?

Reply:
docker step doesn't pass a working-directory so keeping it as false